src/app/components/attestation/attestation-view.jsx

```diff
@@ -128,21 +128,23 @@ function ChplAttestationView(props) {
         </div>
         { attestations.sections
           && (
-            <TableContainer component={Card}>
-              <Table
-                aria-label="Developer Attestations details"
-              >
-                <TableHead>
-                  <TableRow>
-                    <TableCell>Attestation</TableCell>
-                    <TableCell>Response</TableCell>
-                  </TableRow>
-                </TableHead>
-                <TableBody>
-                  { attestations.sections.map((section) => getRows(section, classes)) }
-                </TableBody>
-              </Table>
-            </TableContainer>
+            <Card>
+              <TableContainer component={Card}>
```

Comment:
What's the point of the `Card` component in the TableContainer? If it is supposed to make the table look like a card, why is the enclosing card needed? Does it actually do anything? If not, can it be removed?